eclipse-theia/theia
# core: bump nsfw to ^2.1.2

#### What it does

Commit 1:

- Fix implicit dependency on a Theia extension that prevented me from building...

Commit 2:

- Use the latest NSFW version to fix https://github.com/eclipse-theia/theia/issues/9266 and https://github.com/Axosoft/nsfw/issues/60
- Use the typings from NSFW directly.
- Re-enable file watching testing on Windows.

#### How to test

Everything should work like before, tests should be green.

Windows:

- You can open the file navigator, create files using the terminal and make sure that the file tree updates correctly.

Linux:

- Create nested directories quickly in your workspace (git clone or mkdir -p) and make sure that the file tree updates keep up.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
